Tests/SwiftDocCTests/Rendering/RenderContentCompilerTests.swift

```diff
@@ -131,4 +131,52 @@ class RenderContentCompilerTests: XCTestCase {
             XCTAssertEqual(paragraph, RenderBlockContent.Paragraph(inlineContent: [link]))
         }
     }
+    
+    func testLineBreak() throws {
+        let (bundle, context) = try testBundleAndContext(named: "TestBundle")
+        var compiler = RenderContentCompiler(context: context, bundle: bundle, identifier: ResolvedTopicReference(bundleIdentifier: bundle.identifier, path: "/path", fragment: nil, sourceLanguage: .swift))
+
+        let source = #"""
```

Comment:
Even though this is mostly testing the parsing in Swift Markdown, I think it'd make the test more readable it each example described why it should or shouldn't wrap to two lines. For example
```swift
let source = #"""
Backslash before new line\
is an explicit hard line break.

Two spaces before new line. 
is a hard line break.

Paragraph can't end with hard line break.\

# Headings can't end with hard line break.\

    Code blocks ignore\
    hard line breaks.

A single space before new line 
is a soft line break.
"""#
```

Reply:
Updated